backend/core/views.py

```
@@ -2281,8 +2289,20 @@ def _import_objects(self, parsed_data: dict, domain_name: str):
 
             # Check for missing libraries
             for library in required_libraries:
-                if not LoadedLibrary.objects.filter(urn=library).exists():
-                    missing_libraries.append(library)
+                if not LoadedLibrary.objects.filter(
```

Ruff (0.8.2)

`LoadedLibrary` may be undefined, or defined from star imports

(F405)